### PR TITLE
Removed the yuri command and added a yuri command

### PR DESCRIPTION
**Pull request by MoroseMagician**
Instead of saying "hey" it now actually does a thing, wowie!

### CODE REVIEW

File: commands/booru/yuri.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            throttling: {
+                usages: 1,
+                duration: 10
+            }
+        });
+    }
+
+    async run(message, args) {
+        if (config.search.indexOf(message.channel.id) === -1) return message.reply(`this command only works in search enabled channels.`);
+        safeSearch(('yuri_on_ice ' + args).replace('+','%2B').replace(' ', '+'))
```
Comment (gloss-water) [inline commands/booru/yuri.js]: proper tag is 'yuri!!!_on_ice' unforunately

Comment (gloss-water) [inline commands/booru/yuri.js]: https://safebooru.org/index.php?page=post&s=list&tags=yuri%21%21%21_on_ice
vs
https://safebooru.org/index.php?page=post&s=list&tags=yuri_on_ice

Comment (MoroseMagician) [inline commands/booru/yuri.js]: Oops

**Comment by MoroseMagician** [comment]
it is done :pray:

**Pull request merged**